compiler/include/bison-chapel.h

```diff
@@ -1,19 +1,19 @@
-/* A Bison parser, made by GNU Bison 3.0.4.  */
+/* A Bison parser, made by GNU Bison 2.5.  */
```

Comment:
As far as I know it doesn't matter... I run 'make parser' on chapcs10, which has 2.5. I think you were the last person before then to update it, and you must have used a newer bison.